src/index.html

```diff
@@ -1,12 +1,473 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__li">
+            <a
+              href="#apple"
+              class="nav__a is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#samsung"
+              class="nav__a"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#smartphones"
+              class="nav__a"
+            >
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#pc"
+              class="nav__a"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#gadgets"
+              class="nav__a"
```

Comment:
May be better to use a class name like "nav__link"/"nav__list" (instead - "nav__a"), if the tag then changes, for example.